File: AllSpiceSharp/Models/Favorite.cs
namespace AllSpiceSharp.Models;

public class Favorite : IRepoItem<int>
{
  public int Id { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
  public string AccountId { get; set; }
  public int RecipeId { get; set; }
  public Recipe Recipe { get; set; }
}

File: AllSpiceSharp.client/src/models/Favorite.js
export class Favorite {
  constructor(data) {
    this.id = data.id
    this.createdAt = data.createdAt
    this.updatedAt = data.updatedAt
    this.accountId = data.accountId
    this.recipeId = data.recipeId
    this.recipe = data.recipe
  }
}